[DOCS] JDBC Driver troubleshooting params (#93858)
elastic/elasticsearch: docs/reference/sql/endpoints/jdbc.asciidoc

`allow.partial.search.results` (default `false`):: Whether to return partial results in case of shard failure or fail the query throwing
the underlying exception (default).

[discrete]
==== Troubleshooting

> `debug` (default `false`):: Whether to produce debug logs or not.

Reviewer: ```suggestion
`debug` (default `false`):: Setting it to `true` will enable the debug logging.
```

> `debug.out` (default `out`):: The destination of the debug logs produced by the JDBC Driver. By default, they are sent to standard output. The setting can be also a file path.

Reviewer: From the [source](https://github.com/elastic/elasticsearch/blob/7a2266908d44c303b09e22beb8dc9551b3050c7d/x-pack/plugin/sql/jdbc/src/main/java/org/elasticsearch/xpack/sql/jdbc/JdbcConfiguration.java#L48):

```suggestion
`debug.output` (default `err`):: The destination of the debug logs. By default, they are sent to standard error. Value `out` will redirect the logging to standard output. A file path can also be specified.
```
